src/simulation/BenchmarkTests.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "simulation/Benchmark.h"
+#include "history/HistoryArchive.h"
+#include "lib/catch.hpp"
+#include "simulation/Benchmark.h"
+#include "test/test.h"
+#include "util/Logging.h"
+#include "util/Timer.h"
+#include "util/make_unique.h"
+#include <iostream>
+#include <memory>
+
+using namespace stellar;
+
+const char* LOGGER_ID = "Benchmark";
+
+std::unique_ptr<Benchmark>
+initializeBenchmark(Application& app)
+{
+    Benchmark::BenchmarkBuilder builder{app.getNetworkID()};
+    builder.initializeBenchmark();
+    return builder.createBenchmark(app);
+}
+
+void
+prepareBenchmark(Application& app)
```

Comment:
naming is not very helpful here - what is the distinction between "initialize" and "prepare"?